Access Engineering.js

```diff
@@ -0,0 +1,258 @@
+{
+	"translatorID": "d120a8a7-9d45-446e-8c18-ad9ef0a6bf47",
+	"label": "Access Engineering",
+	"creator": "Vinoth K - highwirepress.com",
+	"target": "^https://www.accessengineeringlibrary.com/content/(book|chapter|case-study|video|calculator|tutorial)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2022-03-30 11:26:17"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020-2021 Vinoth K - highwirepress.com
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if(url.includes('book') && url.match('chapter')) {
+		return 'bookSection';
+	} else if(url.includes('book')) {
+		return 'book';
+	} else if(url.includes('case-study') ||
+			  url.includes('video') ||
+			  url.includes('calculator') ||
+			  url.includes('tutorial')) {
+		return 'webpage';
+	}
+	return false;
+}
+
+function doWeb(doc, url) {
+	scrape(doc, url);
+}
+
+function scrape(doc, url) {
+	let item;
+	let main = doc.querySelector('.page-layout--mhe-layout');
+	
+	switch(detectWeb(doc, url)) {
+	  /* Meta Data: Title, Author, Edition, Publisher,
+				  Date, ISBN, URL, Abstract */
+	  case 'book':
```

Comment:
`case`s should be indented. Run `npm ci && npm run lint -- "Access Engineering.js" --fix` to fix a lot of the style issues automatically.

Reply:
done